lib/hex_web/owners.ex

```diff
@@ -0,0 +1,67 @@
+defmodule HexWeb.Owners do
+  import Ecto
+  alias HexWeb.{User, Package, Repo}
```

Comment:
Do `use HexWeb.Web, :crud` and add that function to `web.ex`. Use the `Web.shared` macro to bring in all aliases.